### PR TITLE
fixed-wording-for-step-5-of-pyramid-project

### LINKED ISSUE #54608: Confusing note/clarification in Step 5 - JavaScript Algorithms and Data Structures (Beta) - Building a Pyramid Generator
### Describe the Issue

Currently on step number 5 there's a note that reads 
>Note that when reassigning a variable that has already been declared, you do not use the let keyword.

As it stands this note is incorrect and unnecessary.

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-introductory-javascript-by-building-a-pyramid-generator/step-5

### Your code

No code, just instructions re-wording.
```



```

### Expected behavior

The note could be changed to:
>Note: You can only re-assign a variables that has been declared with `let`

Or we can simply remove it altogether instead.

### Screenshots

Screen Shot 2024-05-02 at 10 31 38

### System

- Device: [Laptop]
- OS: [MacOS 12.7.4]
- Browser: [Chrome]
- Version: [124.0.6367.91]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by adityarajshekhawat**
Checklist:

-  [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #54608

Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-introductory-javascript-by-building-a-pyramid-generator/step-5

### CODE REVIEW

**Review by jdwilkin4 — COMMENTED** [review]
It looks like you have changed the wrong file.
You should only update the english files.
Please re read through the issue to get the correct file you are supposed to change.

**Comment by adityarajshekhawat** [comment]
> It looks like you have changed the wrong file. You should only update the english files. Please re read through the issue to get the correct file you are supposed to change.

okay i will do that

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660ef31a5be625914a0102cd.md [hunk 1]
```diff
@@ -14,7 +14,8 @@ let programmer = "Naomi";
 programmer = "CamperChan";
 ```
 
-Note that when reassigning a variable that has already been declared, you do **not** use the `let` keyword.
+Note that when reassigning a variable, you do **not** use the `let` keyword again.
+
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660ef31a5be625914a0102cd.md:18]:
```suggestion
```
Keep the empty lines consistent please

File: curriculum/challenges/arabic/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660ef31a5be625914a0102cd.md [hunk 2]
```diff
@@ -13,7 +13,6 @@ When a variable is declared with the `let` keyword, you can <dfn>reassign</dfn>
 let programmer = "Naomi";
 programmer = "CamperChan";
 ```
-
```
Comment (ilenia-magoni) [inline curriculum/challenges/arabic/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660ef31a5be625914a0102cd.md:16]: Please restore the arabic file to the original condition

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]

**Comment by Ksound22** [comment]
We don't make changes to translation files. That's why one of the CIs is failing

File: curriculum/challenges/arabic/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660ef31a5be625914a0102cd.md (hunk 2 above)
Comment (tekluabayneh) [reply curriculum/challenges/arabic/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660ef31a5be625914a0102cd.md:16]: Hi Ilenia-Magoni,

I noticed your contributions to the FreeCodeCamp repo and I'm really impressed with your work! I'm interested in contributing as well but I'm new to the process. Could you share some tips or advice on how I can get started and make meaningful contributions to the project?

I've tried navigating the codebase, but I found it challenging to locate the specific issue folders. Any guidance would be greatly appreciated.

Thank you for your time and help!

Best regards,
Teklu

**Pull request closed without merging**